packages/shared/src/components/post/PostActions.tsx

```diff
@@ -75,50 +95,89 @@ export function PostActions({
     return undefined;
   };
 
+  const toggleDownvote = () => {
```

Comment:
Small suggestion maybe we can modify the "toggleUpvote" to use the same flow? (order of execution)

Also note we probably need the tracking here, we missed it in the DR I think, but it's an essential element.

CC: @idoshamun

Reply:
You're right. Tracking needs to be included and is critical!

Reply:
@capJavert You can find the events here:
https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0

Please let me know if you need some initial help of guidance

Reply:
@rebelchris I will add a separate task so we can merge this into downvote epic so it does not block other tasks since it contains basic logic for downvote.